docs/modules/ROOT/pages/how-tos/exoscale/replace_storage_node.adoc

```diff
@@ -180,54 +180,8 @@ kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pods -w
 
 === Remove old OSD
 
-. Find the OSD on the node you want to replace
-+
-[source,bash]
-----
-OSD_ID=$(kubectl --as=cluster-admin -n syn-rook-ceph-cluster get deploy \
-  -l failure-domain="${NODE_TO_REPLACE}" --no-headers \
-  -o custom-columns="NAME:.metadata.name" | cut -d- -f4)
-echo $OSD_ID
-----
-
-. Verify that we found the correct OSD ID in the previous step
-+
-[source,bash]
-----
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pods -o wide | grep "osd-${OSD_ID}"
-----
-
-. Tell Ceph to take this OSD out of service and relocate data stored on it
-+
-[source,bash]
-----
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
-  ceph osd out "osd.${OSD_ID}"
-----
-
-. Wait for the data to be redistributed ("backfilled") to the other OSDs.
-+
-include::partial$storage-ceph-backfilling.adoc[]
-
-. Remove the OSD from the Ceph cluster
-+
-[source,bash]
-----
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster scale --replicas=0 \
-  "deploy/rook-ceph-osd-${OSD_ID}"
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
-  ceph osd purge "${OSD_ID}" --yes-i-really-mean-it
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
-  ceph osd crush remove "${NODE_TO_REPLACE}"
-----
-
-. Check that the OSD is no longer listed in `ceph osd tree`
-+
-[source,bash]
-----
-kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
-  ceph osd tree
-----
+:osd-node: ${NODE_TO_REPLACE}
```

Comment:
Probably a good idea to move to the top of the page, even if it doesn't affect list numbering here.